BlueprintUI/Tests/ElementStateTests.swift

```diff
@@ -13,6 +13,16 @@ import XCTest
 
 class ElementStateTreeTests: XCTestCase {
 
+    fileprivate var delegate: TestDelegate! = nil
```

Comment:
Fwiw, rather than assigning this in setUp and tearDown, I think it's better to just create it within each test – that avoids more global state, and will eventually make it easier to have multiple cases in a single test method – this is generally the pattern we recommend in SPOS as well – have a fixture or object you create in each test method, vs having instance vars and setting them in setup and teardown.